packages/modal/src/__stories__/getKnobs.js

```diff
@@ -2,16 +2,16 @@ import { action } from "@storybook/addon-actions";
 import { boolean, select, text } from "@storybook/addon-knobs/react";
 import { makeSelectOptions } from "@hig/storybook/utils";
 
-import { types } from "../types";
-
-const typeOptions = makeSelectOptions(types);
+const styleOptions = makeSelectOptions({
+  Standard: "standard",
```

Comment:
This function should be provided with a constant enum, example:
https://github.com/Autodesk/hig/blob/development/packages/banner/src/types.js

Reply:
Addressed by 1c5bfed